Source/Libraries/Adapters/DynamicCalculator/EmailNotifier.cs

```diff
@@ -67,6 +69,17 @@ public class EmailNotifier : DynamicCalculator
 
         #region [ Properties ]
 
+        /// <summary>
```

Comment:
This property description text should match `Description` attribute text except for the "Gets or sets" prefix, so:

```
Gets or sets flag that determines if the trigger continuously being met will send multiple email or should be prevented.
```